.github/workflows/rust.yml

```diff
@@ -160,10 +160,9 @@ jobs:
         cd linera-views
         WASM_BINDGEN_TEST_TIMEOUT=300 wasm-pack test --chrome --headless -- --features web-default
 
-  lint:
+  unexpected-chain-load-operations:
```

Comment:
Should we keep `lint` somewhere in the name? Maybe `lint-unexpected-chain-load-operations`

Reply:
(even better) Shouldn't we group the fast lints together to save git-checkout time?

Reply:
Never mind "checkout" is surprisingly fast.

"Clear some space" is slow, though. In fact, we probably shouldn't do it for linters.